tests/test_exposure.py

```diff
@@ -88,11 +87,6 @@ def setUp(self):
         # test with ExtentI(100, 100) too
         self.exposureCrOnly = afwImage.ExposureF(lsst.geom.ExtentI(100, 100))
 
-        afwImage.Filter.reset()
-        afwImage.FilterProperty.reset()
-
-        defineFilter("g", 470.0)
-
```

Comment:
I'm not sure I understand what's going on here. The test case has assertions that depend on the `g` filter (and I do need those assertions to check backwards-compatibility, so please don't remove them!). However, you've removed the definition of that filter. How does the test still work?